django/contrib/flatpages/admin.py

```diff
@@ -4,15 +4,16 @@
 from django.utils.translation import ugettext_lazy as _
 
 
+@admin.register(FlatPage)
```

Comment:
I suppose it would be nice to remove this from this changeset to keep the diff clean and instead submit a separate cleanup that does this for all admin.py's.